Lesson5/HomeWork.py

```diff
@@ -0,0 +1,106 @@
+"""
```

Comment:
Unfortunately solution is incorrect ( main line):
1) (venv) [antoha@dlhn test_project]$ python ./T-HOMETASKS/L05/l05_h01_YuriiPukich.py
Traceback (most recent call last):
  File "./T-HOMETASKS/L05/l05_h01_YuriiPukich.py", line 71, in <module>
    if all(values[1] >= search_criteria[0],
TypeError: all() takes exactly one argument (3 given)

2) even with fixing this small typo  - you still have gap with conditions check ( pay attention to price condition)

So we have incorrect and not working solution.(

Key moments:
Nothing critical - just be careful next time during task resubmission
- always run your solution and check that script has no errors.
- for provided outout better somehow compare script end expected data. Here we could do it even manually ( only 5 positions)